postman_collection.json

```diff
@@ -570,14 +570,18 @@
 										"exec": [
 											"let jsonResponse = pm.response.json();\r",
 											"\r",
-											"pm.test(\"Response contains only 1 summary\", () => {\r",
-											"    pm.expect(jsonResponse.length).to.eql(1);\r",
+											"pm.test(\"Response contains at least one 1 summary\", () => {\r",
```

Comment:
should be 'Response contains at least 1 summary'

Reply:
Thanks, will fix in a later commit.